conan/tools/env/environment.py

```diff
@@ -58,18 +58,18 @@ def environment_wrap_command(conanfile, env_filenames, env_folder, cmd, subsyste
     if bats:
         launchers = " && ".join('"{}"'.format(b) for b in bats)
         if ps1s:
-            ps1_launchers = f'{powershell} -Command "' + " ; ".join('&\'{}\''.format(f) for f in ps1s) + '"'
-            cmd = cmd.replace('"', "'")
-            return '{} && {} ; cmd /c {}'.format(launchers, ps1_launchers, cmd)
+            ps1_launchers = f'{powershell} -Command "' + " ; ".join('&\'{}\''.format(f) for f in ps1s)
+            cmd = cmd.replace('"', r'\"')
+            return '{} && {} ; cmd /c {}"'.format(launchers, ps1_launchers, cmd)
```

Comment:
Shouldn't this be like this? Aren't we missing a quote there?
```suggestion
            return '{} && {} ; cmd /c "{}"'.format(launchers, ps1_launchers, cmd)
```